apps/opik-backend/src/test/java/com/comet/opik/api/resources/v1/priv/SpansResourceTest.java

```diff
@@ -3189,6 +3190,122 @@ void createWhenTryingToCreateSpanTwice() {
         }
     }
 
+    @Nested
+    @DisplayName("Batch:")
+    @TestInstance(TestInstance.Lifecycle.PER_CLASS)
+    class BatchInsert {
+
+        @Test
+        void batch__whenCreateSpans__thenReturnNoContent() {
```

Comment:
You probably want to attempt to retrieve the new spans and assert that you can get it in some of the happy path tests.

Reply:
done